lib/t/push_notifications/dispatch_job.ex

```diff
@@ -33,17 +33,13 @@ defmodule T.PushNotifications.DispatchJob do
     if match = alive_match(match_id) do
       %Matches.Match{user_id_1: uid1, user_id_2: uid2} = match
 
-      if profile1 = profile_info(uid1) do
-        %Accounts.Profile{name: name1} = profile1
-        data1 = %{"match_id" => match_id, "name" => name1}
-        uid1 |> Accounts.list_apns_devices() |> schedule_apns("match_about_to_expire", data1)
-      end
+      {name1, _gender} = profile_info(uid1)
```

Comment:
"if" didn't need to be deleted:
if profile1 = profile_info(uid1) do
      {name1, _gender} = profile1

Reply:
(same in all other cases below)

Reply:
done